package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "nf start",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint --no-fix --max-warnings 0 src/ bin/ test/ *.js",
+    "lint": "eslint --cache --no-fix --max-warnings 0 --ext .js,.vue src/ bin/ test/ *.js",
```

Comment:
I don't see a `--no-fix` flag for `eslint`. I think it's an option from Vue CLI.